egs/apiai_decode/s5/download-model.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Downlaods Api.ai chain model into exp/api.ai-model/ (will replace one if exists)
+
+DOWNLOAD_URL="https://api.ai/downloads/api.ai-kaldi-asr-model.zip"
+
+echo "Downloading model"
+wget -N $DOWNLOAD_URL || ( echo "Unable to download model: $DOWNLOAD_URL" && exit 1 );
+
+echo "Unpacking model"
+unzip api.ai-kaldi-asr-model.zip || ( echo "Unable to extract api.ai-kaldi-asr-model.zip" && exit 1 );
+
+echo "Moving model to exp/api.ai-model/"
+if [ ! -d exp ]; then
+	mkdir exp;
+fi;
+
+if [ -d exp/api.ai-model ]; then
+	echo "Found existing model, removing";
+	rm -rf exp/api.ai-model/
+fi
+
+mv api.ai-kaldi-asr-model exp/api.ai-model/ || ( echo "Unable to move model to exp/" && exit 1 )
```

Comment:
after exp/api.ai-model/.
IMO, trailing slash is normally used to emphasize when you are moving
something into an already existing directory.


On Mon, Apr 4, 2016 at 5:16 PM, Ilya Platonova <notifications@github.com>
wrote:

> In egs/apiai_decode/s5/download-model.sh
> <https://github.com/kaldi-asr/kaldi/pull/654#discussion_r58450292>:
>
> > +wget -N $DOWNLOAD_URL || ( echo "Unable to download model: $DOWNLOAD_URL" && exit 1 );
> > +
> > +echo "Unpacking model"
> > +unzip api.ai-kaldi-asr-model.zip || ( echo "Unable to extract api.ai-kaldi-asr-model.zip" && exit 1 );
> > +
> > +echo "Moving model to exp/api.ai-model/"
> > +if [ ! -d exp ]; then
> > +	mkdir exp;
> > +fi;
> > +
> > +if [ -d exp/api.ai-model ]; then
> > +	echo "Found existing model, removing";
> > +	rm -rf exp/api.ai-model/
> > +fi
> > +
> > +mv api.ai-kaldi-asr-model exp/api.ai-model/ || ( echo "Unable to move model to exp/" && exit 1 )
>
> Which one of two trailing slashes? :) exp/?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/654/files/7e0f878ee69eedcc5eb06a2d0ad1657c07acb2e9#r58450292>
>